roles/sanitary-sql-access/templates/sanitize-dump.sh.j2

```diff
@@ -1,48 +1,37 @@
 #!/bin/bash
 
-# Script to dump DB, restore it to different schema, run cleaning queries, and dump cleaned schema to a file
+# Create temporary db used to create db dump;
+createdb tmp_sanitary_db;
 
-set -e
```

Comment:
Why would you remove? I use `set -e` to make Clank fail during the [initial run of sanitize-dump script](https://github.com/c-mart/clank/blob/sanitary-sql-access/roles/sanitary-sql-access/tasks/main.yml#L17) if any of these commands exit non-zero.

Reply:
Kinda torn. If the scripts fails, after the temporary db is created, then the script will fail on the next try because `createdb` will exit the script bevcause "db already exists".

Reply:
Aww so awesome learned something new today. See the updated diff.